src/script/components/MessagesList/Message/ContentMessage/ContentMessage.tsx

```diff
@@ -274,6 +266,14 @@ export const ContentMessageComponent = ({
         )}
       </div>
 
+      {[StatusType.FAILED, StatusType.FEDERATION_ERROR].includes(status) && (
+        <CompleteFailureToSendWarning
+          {...(status === StatusType.FEDERATION_ERROR && {unreachableDomain: conversation.domain})}
+          isMessageFocused={msgFocusState}
+          onRetry={() => onRetry(message)}
+        />
+      )}
+
```

Comment:
let's leave this out of this PR, it's not related to Image scrolling at all

This doesn't take into account fed errors
![image](https://github.com/wireapp/wire-webapp/assets/78490891/ed11716e-7a57-4289-b8b5-005be706a235)

Let me take care of this in an other pr

Reply:
got the PR here https://github.com/wireapp/wire-webapp/pull/16943

can you revert https://github.com/wireapp/wire-webapp/pull/16941/commits/4727f522673a5e55774ee070e101f9f73e35a439 ?